### PR TITLE
Custom scrollbar colors

### PR DESCRIPTION
**Pull request by mitchray**
Custom scrollbar styling, before & after

scrollbars

### CODE REVIEW

File: client/stylus/main.styl [hunk 1]
```diff
@@ -26,6 +26,27 @@ a
   &.disabled
    pointer-events none
 
+/* W3C spec */
+* {
```
Comment (AshotN) [inline client/stylus/main.styl:30]: I do not believe you need `*` scope for standard.

File: client/stylus/main.styl [hunk 2]
```diff
@@ -26,6 +26,27 @@ a
...
+  scrollbar-width: thin;
+  scrollbar-color: colorAmber900 colorGray900;
+}
+
+/* Chrome, Edge, and Safari */
+*::-webkit-scrollbar {
```
Comment (AshotN) [inline client/stylus/main.styl:36]: You do not need any scope for WebKit style.

**Review by AshotN — CHANGES_REQUESTED** [review]
Change the scoping for standard compliant to `body`, remove all scoping for webkit style.

**Comment by mitchray** [comment]
On my Windows Firefox 84 the scrollbar-width isn't affecting anything unless its globally `*` , even if I apply it to the elements which have overflow. Could just be something weird on my end but I've left it for now.

**Comment by AshotN** [comment]
> On my Windows Firefox 84 the scrollbar-width isn't affecting anything unless its globally `*` , even if I apply it to the elements which have overflow. Could just be something weird on my end but I've left it for now.

I think I noticed that as well. I’m pretty against scoping with `*`. So we should look into this before this gets merged. 

I do like the custom scroll bars.

**Comment by mitchray** [comment]
If its a performance concern using the ```*``` then I'd say fear not, modern browsers handle it very well

https://www.telerik.com/blogs/css-tip-star-selector-not-that-bad
https://alistapart.com/article/axiomatic-css-and-lobotomized-owls/#section4

Alternatively I can look into 3rd party scrollbar replacements

**Review by AshotN — APPROVED** [review]

**Comment by AshotN** [comment]
Maybe I have an irrational fear of the `*` selector. We can always revisit this once it becomes more standardized.

**Comment by kuzi-moto** [comment]
I don't believe it's an irrational fear. I believe it is against general best practices not to use it. The biggest one being performance, since it gets every single element.

**Comment by AshotN** [comment]
> I don't believe it's an irrational fear. I believe it is against general best practices not to use it. The biggest one being performance, since it gets every single element.

The articles he listed seemed to argue otherwise, I would like to dive deeper into it.

**Pull request merged**